testapi.pm

```diff
@@ -33,7 +33,8 @@ use OpenQA::Isotovideo::NeedleDownloader;
 use Digest::MD5 'md5_base64';
 use Carp qw(cluck croak);
 use MIME::Base64 'decode_base64';
-use Scalar::Util 'looks_like_number';
+use Scalar::Util qw( looks_like_number reftype );
```

Comment:
to stay in line with the existing style please delete the spaces after '(' and before ')'.

https://travis-ci.org/os-autoinst/os-autoinst/builds/571928539#L838 shows you other style issues